docs/product/integrations/notification-incidents/slack/index.mdx

```diff
@@ -56,7 +56,7 @@ Set up an alert rule by going to **Alerts** and clicking "Create Alert". From he
 
 #### Issue Alerts
 
-In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a Slack notification" in the actions dropdown and then select your workspace and channel:
+In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a Slack notification" in the actions dropdown and then select your workspace and channel. You can optionally select any tags to include in the alert as well as other text in the "notes" section. Mentions are supported in the "notes" section - to @ a user, include their user id like `@<USER_ID>`.
```

Comment:
```suggestion
In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a Slack notification" in the actions dropdown. Then, select your workspace and enter your channel. Optionally, you can add tags to include in the notification, as well as other text in the "notes" section. If you want to notify a specific user or team, add a direct mention (for example, "@<USER_ID>") in "notes".
```
"select" implies they can choose from options, but it looks like users need to manually enter in some info